Add documentation for PR Wrangler Shadow program (#32098)
kubernetes/website: content/en/docs/contribute/participate/pr-wranglers.md

The [`fejta-bot`](https://github.com/fejta-bot) bot marks issues as stale after 90 days of inactivity. After 30 more days it marks issues as rotten and closes them.  PR wranglers should close issues after 14-30 days of inactivity.

{{< /note >}}

# PR Wrangler Shadow Program

In late 2021, SIG Docs introduced the PR Wrangler Shadow Program. The program was introduced to help new contributors understand the PR wrangling process.

> ### Become a Shadow

Reviewer: ```suggestion
### Become a shadow
```

> - If you are interested in shadowing as a PR wrangler, please visit the [PR Wranglers Wiki page](https://github.com/kubernetes/website/wiki/PR-Wranglers) to see the PR wrangling schedule for the year and sign up.

Reviewer 2: I feel the suggestion below would clarify that we prepare it on an annual basis.

```suggestion
- If you are interested in shadowing a PR wrangler, please visit the [PR Wranglers Wiki page](https://github.com/kubernetes/website/wiki/PR-Wranglers) to see the PR wrangling schedule for this year and sign up.
```

> - Kubernetes org members can edit the [PR Wranglers Wiki page](https://github.com/kubernetes/website/wiki/PR-Wranglers) to sign up for a week to be shadow a PR Wrangler.

Reviewer 3: ```suggestion
- Kubernetes org members can edit the [PR Wranglers Wiki page](https://github.com/kubernetes/website/wiki/PR-Wranglers) and sign up to shadow an existing PR Wrangler for a week.
```

> - Kubernetes org members can edit the [PR Wranglers Wiki page](https://github.com/kubernetes/website/wiki/PR-Wranglers) and sign up to shadow and existing PR Wrangler for a week.

Reviewer 2: ```suggestion
- Kubernetes org members can edit the [PR Wranglers Wiki page](https://github.com/kubernetes/website/wiki/PR-Wranglers) and sign up to shadow an existing PR Wrangler for a week.
```

> - Others can reach out on [#sig-docs channel](https://kubernetes.slack.com/messages/sig-docs) on Slack to request to shadow a PR Wrangler for a specific week. Feel free to reach out to Brad Topol (`@bradtopol`), Rey Lejano (`@rlejano`),  Natali Vlatko (`@Natali`), Divya Mohan (`@Divya`), Jim Angel (`@jimangel`).

Reviewer 2: Instead of explicitly putting co-chairs' names, I'd rather we redirect them to the leadership section for SIG Docs. I've let Brad's name stay put since he has been the primary POC for this program so far.

```suggestion
- Others can reach out on the [#sig-docs Slack channel](https://kubernetes.slack.com/messages/sig-docs) for requesting to shadow an assigned PR Wrangler. Feel free to reach out to Brad Topol (`@bradtopol`) or one of the  [SIG Docs co-chairs/leads](https://github.com/kubernetes/community/tree/master/sig-docs#leadership).
```

> - Once you've signed up to shadow a PR Wrangler, introduce yourself to the PR Wrangler on Slack.

Reviewer 3: ```suggestion
- Once you've signed up to shadow a PR Wrangler, introduce yourself to the PR Wrangler on the [Kubernetes Slack](slack.k8s.io).
```

The [`fejta-bot`](https://github.com/fejta-bot) bot marks issues as stale after 90 days of inactivity. After 30 more days it marks issues as rotten and closes them.  PR wranglers should close issues after 14-30 days of inactivity.

{{< /note >}}

# PR Wrangler Shadow Program

In late 2021, SIG Docs introduced the PR Wrangler Shadow Program. The program was introduced to help new contributors to understand the PR wrangling process and the PR worklflow.

### Become a Shadow

- If you are interested in participating in the PR wrangler shadow mentoring visit the [PR Wranglers Wiki page](https://github.com/kubernetes/website/wiki/PR-Wranglers) to see the wrangling schedule for the year.

> - You can reach out to Brad Topol (`@bradtopol` on Slack) and register you interest in becoming a shadow for the PR Wrangler role for a particular week.
> 

Reviewer 4: ```suggestion
- You can reach out to Brad Topol (`@bradtopol` on Slack) and register you interest in becoming a shadow for the PR Wrangler role for a particular week.

- Once you've signed up to shadow a PR Wrangler, introduce yourself to the PR Wrangler on Slack

```

### Learning Exceptations

While shadowing a PR wrangler for a particular week, participants will learn:
  
  - The wrangling process which starts when a PR is opened and ends when a PR is merged.
  - Duties of a PR Wrangler.
  - Useful Kubernetes Prow commands.
  - The considerations while applying the `/lgtm` and `/approve` labels.
  - How to interact with people while wrangling a PR.

>   - How to tackle different types of PRs ranging from inactive PRs, PRs needing CLA signing and PRs needing technical reviews for other SIGs.

Reviewer 4: I'm not sure if this section is needed since this page is about PR Wranglers
This PR adds a small section on how to shadow a PR Wrangler hence learn what the PR Wrangler duties are which have already been stated under the **Duties** section

Author: I think we should keep this section because for newcomers it helps them to get a clear idea for what they are going to learn after shadowing (cause I believe every wrangler imparts some knowledge which the _duties_ section does not cover), which will  encourage more people to be a part of the shadow program. Thoughts? 
I am open to all your feedback and will do the needful changes if necessary. Thanks.
